### ISSUE TITLE
[MHV SM / Accessibility] Fieldset provided without a heading

### ISSUE THREAD

**Issue by sarahhortonadhoc** [department-of-veterans-affairs/va.gov-team#124542]
### 📝 User story
As a screen reader user, I want to know the label for a group of inputs, so that I know what I'm being asked to select from the group of inputs.

### 📝 Problem statement
The screen reader announces the full label and description when the user selects a the first input in the group of inputs.

### 📝 Details
<details><summary>Links, screenshots, steps to recreate</summary>

#### 🌌 Supporting screenshots, videos, code snippets

---

##### 🔗 From [Secure Messages > Care teams you recently sent messages to](https://staging.va.gov/my-health/secure-messages/new-message/?prescriptionId=25203727&redirectPath=%2Fmy-health%2Fmedications) username: karsyn.gpsysasynctwo@id.me

<img width="747" height="715" alt="Screenshot of group of radio button inputs showing the heading and description combined as one element" src="https://github.com/user-attachments/assets/14316856-8aa9-4dbf-8d4e-948024e435ef" />

---

#### 👣 Steps to recreate
1. Navigate to Care teams you recently sent messages to](https://staging.va.gov/my-health/secure-messages/new-message/?prescriptionId=25203727&redirectPath=%2Fmy-health%2Fmedications) username: karsyn.gpsysasynctwo@id.me 
2. Notice that the label and description are combined into one element: "Select a team you want to message. This list only includes teams that you’ve sent messages to in the last 6 months. If you want to contact another team, select “A different care team.” (*Required)"
3. Using a screen reader, use the Tab key to navigate to the first radio button input.
4. Notice that the screen reader reads the entire label and description along with the radio button input label.

</details>

---

### 🔗 Point of contact
Sarah Horton (@sarahhortonadhoc)

### ⚠️ Severity level
- 3, Moderate. Should be fixed in 1-3 sprints post-launch.

#### Accessibility Standards
- [Guideline 1.3 Adaptable: Create content that can be presented in different ways (for example simpler layout) without losing information or structure.](https://www.w3.org/WAI/WCAG22/Understanding/adaptable)
  - [1.3.1 Info and Relationships (Level A)](https://www.w3.org/WAI/WCAG22/Understanding/info-and-relationships.html)

#### Experience Standards
- [Category Usability: User is able to easily know or learn how to use the interface to complete tasks](https://design.va.gov/about/experience-standards/#category-usability)
  - [User can identify an element and its state](https://design.va.gov/about/experience-standards/#user-can-identify-an-element-and-its-state)

### 𝌣 Proposed solution or next steps
- Combine the label and required details a heading in the `legend` element: `<h2>Select a team you want to message (*Required)</h2>`
- Leave the remaining helper text inside the `legend` element: "This list only includes teams that you’ve sent messages to in the last 6 months. If you want to contact another team, select “A different care team.”"

### 🔎 Supporting resources
- [Fieldsets, legends, and labels](https://design.va.gov/components/form/#fieldsets-legends-and-labels)
  - [Example 2 - Checkbox group](https://design.va.gov/components/form/#example-2---checkbox-group)

### ✅ Acceptance criteria
- [ ] The system provides a combined heading and required status with the group of radio button inputs.
- [ ] The screen reader announces the heading and required status when the first input has focus.
- [ ] The user can see and hear the group heading and required status when working with the group of radio button inputs.

### ✅ Possible tasks
- [x] Review, refine and acknowledge feedback.
- [x] Update ticket information (assignee, labels, project(s), etc.)
- [ ] Fix and/or document decisions made.
- [ ] Product manager to move ticket in their respective product board to 'QA/A11y Review' after reviewing documented decisions / validating fix.
- [ ] Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

### 🔎 General resources
- USWDS Docs: [USWDS Documentation](https://designsystem.digital.gov/)
- VADS Docs: [VADS Experience Standards](https://design.va.gov/about/experience-standards/)
- VADS Docs: [VADS Components Documentation](https://design.va.gov/components/)
- VADS Docs: [VADS Components Storybook](https://design.va.gov/storybook/?path=/docs/about-introduction--docs)
- VA Platform Docs: [Accessibility defect severity rubric](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric)
- VA Github Docs: [List of common issue tickets](https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/platform/accessibility/common-issues) you can reference, copy, and add to.
- W3C Docs: [Web Content Accessibility Guidelines (WCAG) 2.2](https://www.w3.org/TR/WCAG22/)
- W3C Docs: [WCAG 2.2 Understanding Docs](https://www.w3.org/WAI/WCAG22/Understanding/)

**Comment by steveg-IAT** [comment]
@StacyB2023,
@sarahhortonadhoc found this issue during Stage Review. It's not launch blocking.  We can handle this in our next sprint, OR you can pull it into your project and handle it.  Just let us know.
